hid-keyboard-server/pxt.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "jacdac-hid-keyboard-server",
+    "version": "0.4.63",
+    "description": "MakeCode support for Jacdac keyboard hid server",
+    "dependencies": {
+        "core": "*",
+        "keyboard": "*",
+        "jacdac": "file:../",
```

Comment:
actually on a second look, we should use the github stuff here. the makecode CLI will handle it.